### DISCUSSION TITLE
Java开源课程学习，作者小傅哥(bugstack)

### DISCUSSION THREAD

**Discussion by erbuer** [xu-ux/note#11] [【分享】Share]
**小傅哥的文章质量还是很高的，建议多看看他的专栏！**

贴一个地址：
https://github.com/fuzhengwei/CodeGuide/wiki

相关专栏：

| ID   | Logo                                                         | 专栏文章                                                     | 源码下载                                                     |
| ---- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
| 0    | ![:volcano:](https://camo.githubusercontent.com/6d2f58b376b2e7c0ead8f123b9cabce6732aa0cf7a6f812b686e72ea6e842724/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663330622e706e67) | `网络调试助手`                                               | [NetAssist下载](https://download.csdn.net/download/yao__shun__yu/11835105) |
| 1    | ![:sound:](https://camo.githubusercontent.com/e25a35ce662bcecf115694880b9c740cbb8efa3c4cd023524077febf4a797711/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663530392e706e67) | [`Netty4.x专题`](https://github.com/fuzhengwei/CodeGuide/wiki#sound-Netty4专题) | [itstack-demo-netty](https://github.com/fuzhengwei/itstack-demo-netty) |
| 2    | ![:electric_plug:](https://camo.githubusercontent.com/acb33558773e503b7211d17b61b97c7b445547afb789df27ac0baf37d115482d/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663530632e706e67) | [`手写RPC框架`](https://github.com/fuzhengwei/CodeGuide/wiki#electric_plug-手写RPC框架) | [itstack-demo-rpc](https://github.com/fuzhengwei/itstack-demo-rpc) |
| 3    | ![:computer:](https://camo.githubusercontent.com/e8a0f9d736cbe000d7f0e9f13fe714ec11fc5d65528f2ecc515ecc36beabc65b/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663462622e706e67) | [`用Java实现JVM`](https://github.com/fuzhengwei/CodeGuide/wiki#computer-用Java实现JVM) | [itstack-demo-jvm](https://github.com/fuzhengwei/itstack-demo-jvm) |
| 4    | ![:ghost:](https://camo.githubusercontent.com/dca6ad9826005e1362b04da7c77bfce59c92b32aac727f98430608e15d5bcd35/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663437622e706e67) | [`基于JavaAgent的全链路监控`](https://github.com/fuzhengwei/CodeGuide/wiki#ghost-基于JavaAgent的全链路监控) | [itstack-demo-agent](https://github.com/fuzhengwei/itstack-demo-agent) |
| 5    | ![:shower:](https://camo.githubusercontent.com/1f32035f17fb45d77e872400edb8683285b3c3eb4fe5909d5bf444421149b805/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663662662e706e67) | [`iot-gateway网关案例`](https://github.com/fuzhengwei/CodeGuide/wiki#shower-iot-gateway网关案例) | [itstack-demo-iot-gatewary](https://github.com/fuzhengwei/itstack-demo-iot-gatewary) |
| 6    | ![:triangular_ruler:](https://camo.githubusercontent.com/f92c70c3db0af28bb119c4d45f2752ca2de612895801b0ad74066d12ec6e0184/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663464302e706e67) | [`DDD领域驱动设计落地`](https://github.com/fuzhengwei/CodeGuide/wiki#triangular_ruler-DDD领域驱动设计落地) | [itstack-demo-ddd](https://github.com/fuzhengwei/itstack-demo-ddd) |
| 7    | ![:outbox_tray:](https://camo.githubusercontent.com/f73bf5965ee98819e6b7783815bf04a479367f24c791bbd73f73d83143f3c7da/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663465342e706e67) | [`SpringCloud入门案例`](https://github.com/fuzhengwei/CodeGuide/wiki#outbox_tray-SpringCloud入门案例) | [itstack-demo-springcloud](https://github.com/fuzhengwei/itstack-demo-springcloud) |
| 8    | ![:performing_arts:](https://camo.githubusercontent.com/ab899cf1b6b724d765ccaee37dcba36b87b098780e5ddcca666fcdde30dc8757/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663361642e706e67) | [`微信公众号开发`](https://github.com/fuzhengwei/CodeGuide/wiki#performing_arts-微信公众号开发) | [itstack-ark-wx](https://github.com/fuzhengwei/itstack-ark-wx-test) |
| 9    | ![:nut_and_bolt:](https://camo.githubusercontent.com/9743725936df56ba64bfc6a054b6d4e62e290a14aadf544a1763ab272199a768/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663532392e706e67) | [`SpringBoot中间件开发`](https://github.com/fuzhengwei/CodeGuide/wiki#nut_and_bolt-SpringBoot中间件开发) | [door-spring-boot-starter](https://github.com/fuzhengwei/door-spring-boot-starter) |
| 10   | ![:art:](https://camo.githubusercontent.com/cb47df9902533d65b5ffab51e79f380c88caca1b2a7ff42476cb64591a83ae63/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663361382e706e67) | [`服务框架搭建`](https://github.com/fuzhengwei/CodeGuide/wiki#art-服务框架搭建) | [itstack-demo-frame](https://github.com/fuzhengwei/itstack-demo-frame) |
| 11   | ![:flashlight:](https://camo.githubusercontent.com/4d9da63d23fc4e3e07d1a8942d6e395432a62df0b2a59f944ea18900bc293522/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663532362e706e67) | [`源码分析(Spring、Mybatis、Schedule)`](https://github.com/fuzhengwei/CodeGuide/wiki#flashlight-源码分析) | [itstack-demo-code](https://github.com/fuzhengwei/itstack-demo-code) |
| 12   | ![:airplane:](https://camo.githubusercontent.com/72d8cd8f294665813c116fee31a9901c79655b2b49a060af79af7c603e10f6b9/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f323730382e706e67) | [`Drools规则引擎`](https://github.com/fuzhengwei/CodeGuide/wiki#airplane-Drools规则引擎) | [itstack-demo-drools](https://github.com/fuzhengwei/itstack-demo-drools) |
| 13   | ![:tractor:](https://camo.githubusercontent.com/d8cd40348c552ec435e50a0be7f357c135be8bd1da93359bc9154003debd4d5f/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663639632e706e67) | [`ASM字节码编程`](https://github.com/fuzhengwei/CodeGuide/wiki#tractor-ASM字节码编程) | [itstack-demo-asm](https://github.com/fuzhengwei/itstack-demo-asm)、[SQM](https://github.com/WormholePistachio/SQM) |
| 14   | ![:paw_prints:](https://camo.githubusercontent.com/58e77f786808b4a9b4b7d998c21284280c4603e50c376b1726bc7b7005f38701/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663433652e706e67) | [`我的大学四年到毕业工作5年的学习资源和面试汇总`](https://github.com/fuzhengwei/CodeGuide/wiki#paw_prints-我的大学四年到毕业工作5年的学习路线资源和面试汇总) | [https://pan.baidu.com/s/4mmX7sDy - （if链接失效，加我微信：fustack）](https://pan.baidu.com/s/4mmX7sDy) |
| 15   | ![:walking:](https://camo.githubusercontent.com/d61167739318c51cef8f64d8e4b8912237add9d92de1b65f155ed059f6fa234a/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663662362e706e67) | [`Netty+JavaFx实战：仿桌面版微信聊天`](https://github.com/fuzhengwei/CodeGuide/wiki#walking-Netty仿桌面版微信聊天) | [NaiveChat](https://github.com/fuzhengwei/NaiveChat)         |
| 16   | ![:bookmark_tabs:](https://camo.githubusercontent.com/efc0cb61510f3614387f55ca794dfa3068e790cad8649c177c9a542ed73b7215/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663464312e706e67) | [`JDK1.8新特性41个案例讲解`](https://bugstack.cn/itstack-demo-any/2019/12/10/有点干货-Jdk1.8新特性实战篇(41个案例).html) | [itstack-demo-jdk8](https://github.com/fuzhengwei/itstack-demo-jdk8) |
| 17   | ![:bike:](https://camo.githubusercontent.com/14031e967b9e57d760b4490016b69fef4ca7aee1f2c65edb58b75494d46e767b/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663662322e706e67) | [`小傅哥的《字节码编程》专栏`](https://github.com/fuzhengwei/CodeGuide/wiki#bike-字节码编程专栏) | [itstack-demo-bytecode](https://github.com/fuzhengwei/itstack-demo-bytecode) |
| 18   | ![:school_satchel:](https://camo.githubusercontent.com/eb60ae4339fbdecaaa77b6975005dbb00d251ec7ba9e988def99cbd020c21e1a/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663339322e706e67) | [`重学Java设计模式「真实场景实战」`](https://github.com/fuzhengwei/CodeGuide/wiki#school_satchel-实战设计模式) | [itstack-demo-design](https://github.com/fuzhengwei/itstack-demo-design) |
| 19   | ![:bookmark_tabs:](https://camo.githubusercontent.com/efc0cb61510f3614387f55ca794dfa3068e790cad8649c177c9a542ed73b7215/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663464312e706e67) | [`面经手册`](https://github.com/fuzhengwei/CodeGuide/wiki#bookmark_tabs-面经手册) | [interview](https://github.com/fuzhengwei/interview)         |
| 20   | ![:racehorse:](https://camo.githubusercontent.com/cf54ceb21d41967000563111f82308b47925a5b4c3de30dd15b0c4dd6371d666/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663430652e706e67) | [`码场故事`](https://github.com/fuzhengwei/CodeGuide/wiki#racehorse-码场故事) | -                                                            |
| 21   | ![:factory:](https://camo.githubusercontent.com/c6b3494a5c9d0c488f0e2a9eebff350a85b5c4c947c782f3da96f374b9691046/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663365642e706e67) | [`架构设计`](https://github.com/fuzhengwei/CodeGuide/wiki#factory-架构设计) | -                                                            |
| 22   | ![:seedling:](https://camo.githubusercontent.com/6e6dc5f22dc6eb92a69798598671de610ded65f3e6b364d90dc0239d6d8cc358/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f31663333312e706e67) | [`Spring手撸专栏`](https://github.com/fuzhengwei/CodeGuide/wiki#seedling-Spring手撸专栏) | [small-spring](https://github.com/fuzhengwei/small-spring)   |
| 23   | ![:cloud:](https://camo.githubusercontent.com/ba5bdec2e1697a2120bf83f4043ea192538066104568e2dbc9b476ded85791bf/68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f69636f6e732f656d6f6a692f756e69636f64652f323630312e706e67) | [`云服务学习`](https://github.com/fuzhengwei/CodeGuide/wiki#cloud-云服务学习) | [云服务学习](https://www.yuque.com/fuzhengwei/hzgecg/hvoy2a) |

![图片](https://user-images.githubusercontent.com/30521906/126886843-ecc73c09-5464-4282-b430-7b022d831863.png)

**Comment by xu-ux** [comment]
质量很高，已经在看了 @erbuer
